README.md

```diff
@@ -23,9 +29,18 @@ system config:
 }
 ```
 
-In reality, my package overrides are split across multiple files, since I have
-more than just this one, but the idea is the same. It should also be possible to
-set this up with a user-local nixpkgs config.
+#### Local User Setup
+
+You can also set this up specifically for a single user by adding it to that
+user's nix configuration, located at `/home/%user%/.nixpkgs/config.nix`.
```

Comment:
I think saying `$HOME/.nixpkgs/config.nix` or `~/.nixpkgs/config.nix` would be more idiomatic.